src/postgres/commands/PostgresAccountWizard/PostgresServerCredPWStep.ts

```diff
@@ -13,11 +13,12 @@ export class PostgresServerCredPWStep extends AzureWizardPromptStep<IPostgresWiz
 
     public async prompt(wizardContext: IPostgresWizardContext): Promise<void> {
         const user = nonNullProp(wizardContext, 'adminUser');
+        const pwConditionMsg = localize('passwordConditionMsg', 'Password must contain characters from three of the following categories - uppercase letters, lowercase letters, numbers (0-9), and non-alphanumeric characters (!, $, etc.).');
```

Comment:
nit: we usually don't pass string constants as parameters. You can just move the string to the class or file level so that it's in scope for both usages.